### PR TITLE
More py3 cleanups

### PR DESCRIPTION
**Pull request by palango**
More small cleanups for python 3.

- Fix a type annotation in client
- use `super()` instead of `super(Type, self)`

### CODE REVIEW

File: raiden/network/rpc/client.py [hunk 1]
```diff
@@ -680,7 +680,7 @@ def eth_estimateGas(
             to: address = b'',
             value: int = 0,
             data: bytes = b'',
-            startgas: Optional[int] = None) -> int:
+            startgas: Optional[int] = None) -> Optional[int]:
```
Comment (hackaugusto) [inline raiden/network/rpc/client.py]: btw, you don't need to declare `Optional[int]` if the default value is `None`, it's inferred for you:

`startgas: Optional[int] = None) -> Optional[int]:`

is the same as:

`startgas: int = None) -> Optional[int]:`

Additionally, please use the format we aggreed on:

```
        startgas: Optional[int] = None
) -> Optional[int]:
```

Comment (palango) [reply raiden/network/rpc/client.py]: Nice, didn't know about that.

Comment (hackaugusto) [reply raiden/network/rpc/client.py]: Me neither, I literraly learned it half an hour ago :) 

https://www.python.org/dev/peps/pep-0484/#union-types

Comment (LefterisJP) [reply raiden/network/rpc/client.py]: That's cool!

**Pull request merged**